sceptre/cli/list.py

```diff
@@ -89,6 +89,21 @@ def list_outputs(ctx, path, export):
                         ),
                         "text",
                     )
+
+    elif export == "stackoutput":
+        stack_names = {stack.name: stack.external_name for stack in plan.graph}
+        for response in responses:
+            for stack_name, stack in response.items():
+                for output in stack:
+                    write(
+                        "!stack_output_external {0}::{1} [{2}]".format(
```

Comment:
Not clear from this diff, but use of format rather than f-string is consistent with the implementation of the case `export == "envvar"`.